### DISCUSSION TITLE
It is possible to check string with "Contains" operator in Check block?

### DISCUSSION THREAD

**Discussion by lHuKKal** [postmanlabs/postman-flows#400] [General]
Hello!

I have one Patch request with one field, but this field is unique and i need to check this field for the presence of certain text in the Check block. Is there any way to check this text, for example with the operator "contains"? I just couldn't find it in the Check block?

**Comment by samay-v** [comment]
You can use the `in` operator in the check block to achieve this 👍🏽

**Reply by lHuKKal** [reply]
Hello!

Thanks for reply, but i've tested IN operator and it doesn't work like "contains" operator. Maybe I am doing something wrong? (see attached screenshot Example)

![image](https://user-images.githubusercontent.com/97214688/195012531-23745cd0-ae15-45e5-968c-1c558cb0a2fc.png)

In example, i check subcategory name with the IN (contains) operator, but this check fails
